src/parachain/amm.ts

```diff
@@ -176,7 +198,7 @@ export class DefaultAMMAPI implements AMMAPI {
     private async _getStableLpTokens(): Promise<Array<StableLpToken>> {
         const stablePools = await this.api.query.zenlinkStableAmm.pools.entries();
         const stableLpTokens = stablePools.map(([key, poolData]) => {
-            const poolBase = DefaultAMMAPI.getStableBasePool(poolData.unwrap());
+            const poolBase = DefaultAMMAPI.getStablePoolInfo(poolData.unwrap());
```

Comment:
I know this is pre-existing, but noticed the unwrap before checking.
Are we sure that we don't need to filter for `poolData.isSome` ?

Reply:
thanks, I missed it here :)